f5-distributed-cloud/README.md

```diff
@@ -0,0 +1,92 @@
+## Overview
+
+F5 Distributed Cloud (XC) Services provides a global cloud native platform where customers can deploy, manage, and secure their applications regardless of whether the application resides in a public cloud, in a private data center, or a colocation. It provides a variety of both ADN and CDN services.
+
+The F5 XC platform includes the Global Log Receiver. F5 XC global log receiver can be configured to securely send logs to a Datadog HTTPS logging endpoint. Configuration can be done via the [F5 XC Console UI][6].
+
+
+The F5 Distributed Cloud Services integration includes:
+
+- Dashboard - *Access Log Overview*
+- Saved View - *Including facets for commonly queried fields*
+
+## Setup
+
+You can configure global log streaming in either system namespace or in shared namespace. In case of configuring in shared namespace, you can configure to stream from either shared namespace or all namespaces or specific list of namespaces. If you are configuring in system namespace, you can only stream logs from system namespace.
+
+The example shown in this guide creates a global log receiver object in the Console in system namespace for streaming the logs to the external log collection system.
+
+Perform the following in the F5® Distributed Cloud Console:
+
+**Step 1: Start creating a global log receiver**
+
+- In the Console home page, select Shared Configuration service.
+- Select Manage > Global Log Receiver.
+- Select Global Log Receiver in case of Cloud and Edge Sites service.
+
+![snapshot][3]
```

Comment:
```suggestion
**Step 1: To create a global log receiver**

1. Within the F5® Distributed Cloud Console, navigate to Shared Configuration service.
2. Select Manage > Global Log Receiver.
3. Select Global Log Receiver in case of Cloud and Edge Sites service.
4. Click add Global Log Receiver button

![snapshot][3]
```